mappings/net/minecraft/util/registry/Registry.mapping

```diff
@@ -1,4 +1,147 @@
 CLASS net/minecraft/class_2378 net/minecraft/util/registry/Registry
+	COMMENT A registry is used to register various in-game components. Almost all parts of the
+	COMMENT game - from blocks, items, and entity types, to cat types, goat horn instruments,
+	COMMENT and structure pools - are registered in registries. Registry system allows the game
+	COMMENT to enumerate all known types of something, and to assign a unique identifier to each
+	COMMENT of those. Therefore, registering an object in the registry plays a very important
+	COMMENT role, and failure to register new instances of registerable object usually results
```

Comment:
I recall registry is mainly important for its roles in serializing data in saves/over networks, where it establishes integer and string ids to object mappings. Is that a better description on its importance?

Reply:
In modern Minecraft raw ID is usually not used, even in networks.

Reply:
That too, but not all registries use that - enumeration is the main function.